docs/ui/toolbar.md

```diff
@@ -1,41 +1,107 @@
 # The Toolbar
 
-The toolbar contains frequently used commands, such as saving and sharing, your current position within the dataset, and the ability to switch between various modes for viewing. Further, it provides access to all the tools for annotation, navigation, and more.
+The toolbar is your primary interface for interacting with WEBKNOSSOS. Located at the top of the screen, it provides quick access to essential functions for viewing, annotating, and sharing your work. This guide explains each component and its purpose in your workflow.
```

Comment:
might be nitpicky, but for me the primary interface are the data viewports + mouse :see_no_evil: but I dont want to be päbstlicher als der pabst.

Reply:
:+1: